Source/Scene/ModelExperimental/ModelExperimentalSceneNode.js

```diff
@@ -0,0 +1,27 @@
+export default function ModelSceneNode(options) {
+  /**
+   * @type {ModelComponents.Node}
+   */
+  this._node = options.node;
+
+  /**
+   * @type {Matrix4}
+   */
+  this._modelMatrix = options.modelMatrix;
+
+  /**
+   * Pipeline stages to apply across the mesh primitives of this node.
+   */
+  this._pipelineStages = [];
+
+  /**
+   * @type {ModelExperimentalSceneMeshPrimitive}
+   */
+  this._sceneMeshPrimitives = [];
+
+  initialize(this);
+}
+
+function initialize(sceneNode) {
```

Comment:
@sanjeetsuhag I presume this function is empty because it won't be until instancing that you fill this out?

Reply:
Correct.

Reply:
cool, I put a comment in the code to make this clear